cloudify_gcp/utils.py

```diff
@@ -199,6 +201,73 @@ def resource_created(ctx, resource_field):
     return False
 
 
+def resource_started(ctx, resource):
+    try:
+        resource_status = resource.get().get('status')
+    except HttpError as e:
+        if e.resp.status == http_client.NOT_FOUND:
+            resource_status = None
+        else:
+            raise e
+
+    if not resource_status:
+        ctx.operation.retry(
+            'Kubernetes resource is still provisioning', 15)
```

Comment:
Is `resource_status` `None` really a retry-able state? Isn't it an error? Or should we try to create again? Because during delete you assume it has been deleted. Not sure I understand this logic.

Reply:
it based on:
```
    created_node = get_node(node_pool)
    if not created_node:
        ctx.operation.retry(
            'Kubernetes node pool {0} '
            'is still provisioning'.format(name), 15)
```

If we have not found object - we go to next round.

Reply:
I have removed such code: b61cac6